main/main.tex

```diff
@@ -32,19 +35,24 @@
 \editor{Associate Editor: XXXXXXX}
 
 \abstract{
-\textbf{Motivation:} Obtaining high-quality data from mass spectrometry (MS) 
-experiments can be challenging due to various factors that can impact the 
-accuracy and reproducibility of the data. Quality control (QC) 
-techniques are needed to guarantee that the data sets are 
-of high quality.\\
-\textbf{Results:} The \texttt{MsQuality} R-package calculates and assesses 
-various quality metrics for mass spectrometry-derived spectral data at the 
-per-measurement level. It calculates around 40 low-level quality metrics based on 
-the controlled vocabulary of the mzQC quality metrics defined by HUPO-PSI. 
-The package helps to identify low-quality measurements and track data quality, 
-ultimately improving the quality and reproducibility of mass spectrometry 
-data for robust scientific discoveries. \\
-\textbf{Availability:} MsQuality is implemented in R and is available through 
+\textbf{Motivation:} Multiple factors can impact accuracy and reproducibility
+of mass spectrometry data. There is a need to integrate quality
+assessment and control into data analytic workflows. \\
+\textbf{Results:} The \texttt{MsQuality} package calculates % and assesses 
+% various quality metrics for mass spectrometry-derived spectral data at the 
+% per-measurement level. It calculates
```

Comment:
measurement refers e.g. to one chromatographic run or run by Flow injection analysis. it could also refer to one plex in labelled MS. @jorainer do you have any better term for this or can think of a way of avoiding this term in the abstract without giving a lengthy definition?

Reply:
I see the point. I sometimes use the term "injection" for a measurement of a single sample - which would then cover both the LC-MS based measurement as well as the FIA you mentioned. But the general audience for Bioinformatics will not be familiar with this term...

What about keeping the changes from Wolfgang and simply add

> ... defined by HUPO-PSI on a single mass spectrometry-based measurement of a sample.

Reply:
Agreed, as you and Wolfgang suggested